lib/Providers/Qr/EndroidQrCodeProvider.php

```diff
@@ -57,17 +62,17 @@ protected function qrCodeInstance($qrtext, $size)
         return $qrCode;
     }
 
-    private function handleColor($color)
+    private function handleColor(string $color): Color
     {
         $split = str_split($color, 2);
         $r = hexdec($split[0]);
         $g = hexdec($split[1]);
         $b = hexdec($split[2]);
 
-        return $this->endroid4 ? new Color($r, $g, $b, 0) : ['r' => $r, 'g' => $g, 'b' => $b, 'a' => 0];
+        return $this->endroid4 ? new Color($r, $g, $b, 0) : array('r' => $r, 'g' => $g, 'b' => $b, 'a' => 0);
```

Comment:
`[]` is not a "newer" syntax. It is a different syntax. I prefer to keep the `array()` syntax because:

* it was like this before
* it is much more readable IMHO